api/data_refinery_api/views.py

```diff
@@ -377,9 +377,26 @@ class CreateDatasetView(generics.CreateAPIView):
 @method_decorator(name='put', decorator=swagger_auto_schema(operation_description="""
 Modify an existing Dataset.
 
-Set `start` to `true` along with a valid activated API token (from `/token/`) to begin smashing and delivery.
+In order to begin smashing, you have to provide an activated api token. The token can be
+requested with [/token](#tag/token), and has to be sent in the header of the request
+with the key: `API-TOKEN`. Example:
```

Comment:
I think I realized what seems weird to me about this phrasing. It's the use of `key`. I don't think I've ever heard `key` used when talking about HTTP headers. I've generally heard each field referred to as a header, but I think that might also be slightly incorrect technically. I think the term we should use is "header field", as in "must be provided in the `API-KEY` header field on the request."

I also think that instead of saying `The token can be requested with [/token](#tag/token)...`, we should just say something like "An activated API key must be provided in the `API-KEY` header field on the request. To acquire and activate an API key see the documentation for the `/token` endpoint. The reason for this change is that if someone doesn't actually go look at the documentation for the `/token` endpoint then they might end up trying an inactivated key.